.eslintrc.js

```diff
@@ -0,0 +1,170 @@
+module.exports = {
+    env: {
+        browser: true,
+        commonjs: true,
+        es2021: true,
+        jest: true,
+        node: true
+    },
+    extends: [
+        "eslint:recommended",
+        require.resolve('eslint-config-standard'),
+        "plugin:react/recommended",
+        "plugin:@typescript-eslint/recommended",
+        'plugin:@typescript-eslint/recommended-requiring-type-checking'
+    ],
+    ignorePatterns: [
+        '.eslintrc.js',
+        '.github/**',
+        '.vscode/**',
+        '.yarn/**',
+        '**/build/*',
+        '**/coverage/*',
+        '**/node_modules/*'
+    ],
+    overrides: [{
+        files: ['*.js', '*.cjs', '*.mjs'],
+        rules: {
+            '@typescript-eslint/explicit-function-return-type': 'off',
+            '@typescript-eslint/no-var-requires': 'off',
+            '@typescript-eslint/no-unsafe-argument': 'off',
+            '@typescript-eslint/no-unsafe-assignment': 'off', //тоже ничего не делает?
+            '@typescript-eslint/no-unsafe-call': 'off',
+            '@typescript-eslint/no-unsafe-member-access': 'off',
+            '@typescript-eslint/no-unsafe-return': 'off',// ничего не делает?
+            '@typescript-eslint/restrict-plus-operands': 'off',
+            '@typescript-eslint/restrict-template-expressions': 'off',
+            "sort-keys-fix/sort-keys-fix": "warn" // сама добавила для автоматической сортировки ключей
```

Comment:
it's part of js rule, this one should be higher (not in "exceptions for js files")